job-server-python/src/python/run-tests.cmd

```diff
@@ -15,7 +15,8 @@ setlocal
 cd %~dp0
 
 set PYTHONPATH=%~dp0;%SPARK_HOME%\python\lib\pyspark.zip;%SPARK_HOME%\python\lib\py4j-0.9-src.zip;%PYTHONPATH%
-python test\apitests.py
+set PYSPARK_PYTHON=%1
+%1 test\apitests.py
```

Comment:
instead of `%1`, you can use `PYSPARK_PYTHON`

Reply:
Done